internal/endpoint.go

```diff
@@ -1,8 +1,50 @@
 package internal
 
-type Endpoint string
+import (
+	"os"
 
-const (
-	API   Endpoint = "https://api.omise.co"
-	Vault          = "https://vault.omise.co"
+	"github.com/joho/godotenv"
```

Comment:
I wouldn't include `joho/godotenv` here, but rather leave it to the application.